packages/docutils/lib/mkdocs.js

```diff
@@ -22,9 +22,9 @@ export async function verifyMkdocs() {
 /**
  * Run 'mkdocs build' on a project
  *
- * @param {string} configPath path to mkdocs config yml
- * @param {string} outputDir directory mkdocs should build into
- * @param {?string} theme theme name
+ * @param {string} configPath - path to mkdocs config yml
+ * @param {string} outputDir - directory mkdocs should build into
+ * @param {?string} theme - theme name
```

Comment:
The previous syntax is invalid.

```suggestion
 * @param {string?} theme - theme name
```